src/elements/content-sidebar/activity-feed/task/Task.js

```diff
@@ -97,59 +110,40 @@ class Task extends React.Component<Props> {
                                   .map(fillUserPlaceholder)
+                                  .slice(0, MAX_AVATARS)
                                   .map(({ id: assignmentId, assigned_to, status }) => {
-                                      switch (status) {
-                                          case TASK_COMPLETED:
-                                          case TASK_APPROVED:
-                                              return <CompletedAssignment {...assigned_to} key={assigned_to.id} />;
-                                          case TASK_REJECTED:
-                                              return <RejectedAssignment {...assigned_to} key={assigned_to.id} />;
-                                          case TASK_INCOMPLETE:
-                                              return (
-                                                  <PendingAssignment
-                                                      {...assigned_to}
-                                                      key={assigned_to.id}
-                                                      onTaskApproval={
-                                                          isPending
-                                                              ? noop
-                                                              : () =>
-                                                                    onAssignmentUpdate(id, assignmentId, TASK_APPROVED)
-                                                      }
-                                                      onTaskReject={
-                                                          isPending
-                                                              ? noop
-                                                              : () =>
-                                                                    onAssignmentUpdate(id, assignmentId, TASK_REJECTED)
-                                                      }
-                                                      shouldShowActions={
-                                                          onAssignmentUpdate !== noop &&
-                                                          currentUser &&
-                                                          assigned_to.id === currentUser.id
-                                                      }
-                                                  />
-                                              );
-                                          default:
-                                              return null;
-                                      }
+                                      return <AssigneeStatus key={assignmentId} status={status} user={assigned_to} />;
                                   })
                             : null}
+                        {hiddenAssigneeCount > 0 ? (
+                            <span className="bcs-task-assignment-avatar bcs-task-assignment-count">
```

Comment:
Does this need an `aria-label` or `title` to explain its meaning?

Reply:
TODO: add tooltip